README.md

```diff
@@ -3,7 +3,31 @@
 More information on this service can be found on Confluence:
 
 * https://confluence.ons.gov.uk/display/SDC/SDS
+---
+## Dockerized
+The docker-compose will launch the SDS application, two storgae emulators(firebase and bucket) and two cloud functions. The SDS application will also support hot reloading within the `/src/app` directory.
 
+- You will need to create a new file called `mock_google_app_key.json` within the `/devtools` directory and copy the contents from this fake service acount found here [Mock service account](https://github.com/firebase/firebase-admin-python/blob/master/tests/data/service_account.json).
+
+This will launch the SDS application and two emulators, the SDS application will support hot reloading within the `/src/app` directory. Start the containers:
+
+```
+docker-compose up
+```
+Once loaded you can then utilise the following support tools.
```

Comment:
I would re-write this slightly to say "Once, loaded you can do the following:", because "support tools" doesn't sound right to me. Then each bullet point would start with what you can do (in bold if you like), for example:

* View the OpenAPI spec docs
* See the files put into cloud storage
* Visualise the contents of the database

... I don't really understand what "thundercloud" is?

Reply:
re-worded all of this and included the curl command, we can go over this.

Reply:
If you can, please can you have another look at my comments and see if you agree with the suggested changes? If not, then we can leave it as is.